bundles/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/handler/ModbusPollerThingHandler.java

```diff
@@ -246,7 +244,7 @@ public synchronized void initialize() {
             if (!ModbusBindingConstantsInternal.READ_FUNCTION_CODES.containsKey(type)) {
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
                         String.format("No function code found for type='%s'. Was expecting one of: %s", type,
-                                StringUtils.join(SORTED_READ_FUNCTION_CODES, ", ")));
+                                String.join(", ", SORTED_READ_FUNCTION_CODES)));
```

Comment:
String join also works on lists so it should not be needed to use native arrays?

Reply:
True, you are right. I reverted to List<String> again.